### ISSUE TITLE
Cannot read property 'wrapAwait' of undefined

### ISSUE THREAD

**Issue by dariusriggins** [babel/babel#4593]
```js
class MyFunClass {
    async doWork(){
        const response = await this.client.get('stuff');
    }
}
```

```js
{
    "presets": ["es2015-node5", "react"],
    "plugins": ["transform-async-to-generator"]
}
```

### Expected Behavior
Was expecting to compile without errors, instead got the following error running via `babel src -d lib`.

```
TypeError: src/client/actions/InstallActionHelper.js: Cannot read property 'wrapAwait' of undefined
    at exports.default (/Users/darius/co/projpath/node_modules/babel-helper-remap-async-to-generator/lib/index.js:8:23)
    at PluginPass.Function (/Users/darius/co/projpath/node_modules/babel-plugin-transform-async-to-generator/lib/index.js:13:80)
    at newFn (/Users/darius/co/projpath/node_modules/babel-traverse/lib/visitors.js:276:21)
    at NodePath._call (/Users/darius/co/projpath/node_modules/babel-traverse/lib/path/context.js:76:18)
    at NodePath.call (/Users/darius/co/projpath/node_modules/babel-traverse/lib/path/context.js:48:17)
    at NodePath.visit (/Users/darius/co/projpath/node_modules/babel-traverse/lib/path/context.js:105:12)
    at TraversalContext.visitQueue (/Users/darius/co/projpath/node_modules/babel-traverse/lib/context.js:150:16)
    at TraversalContext.visitSingle (/Users/darius/co/projpath/node_modules/babel-traverse/lib/context.js:108:19)
    at TraversalContext.visit (/Users/darius/co/projpath/node_modules/babel-traverse/lib/context.js:192:19)
    at Function.traverse.node (/Users/darius/co/projpath/node_modules/babel-traverse/lib/index.js:114:17)
```

### Your Environment

| software         | version
| ---------------- | -------
| Babel            |  6.9.1 requested, deps bringin in 6.16.0 for babel-helper-remap-async-to-generator
| node             | 5.11.0
| npm              | 3.8.6
| Operating System | OSX & Ubuntu

**Comment by hzoo** [comment]
6.9.1 of babel-cli? Can you update to the latest version for everything?

```js
{
  "babel": {
    "presets": [
      "es2015-node5",
      "react"
    ],
    "plugins": [
      "transform-async-to-generator"
    ]
  },
  "devDependencies": {
    "babel-cli": "^6.16.0",
    "babel-plugin-transform-async-to-generator": "^6.16.0",
    "babel-preset-es2015-node5": "^1.2.0",
    "babel-preset-react": "^6.16.0"
  },
  "scripts": {
    "start": "babel a.js"
  }
}
```

which the code snippet posted compiles without error.

**Comment by dariusriggins** [comment]
Upgrading to latest everything did fix that error for me. Thanks.

**Comment by hillerstorm** [comment]
Sure upgrading to the latest version works, but doing breaking changes to packages like this while still having a semver range (^) specified is not a good habit :/

(I'm referring to the change in babel-helper-remap-async-to-generator that changed the default export)

You should either stick to exact versions in all of your packages here on babel or change to proper semver :(
Regressions like this should be caught by your tests, not by us trying to deploy our applications.

**Comment by danez** [comment]
@hillerstorm We do proper semver,except for small spec compliancies which are treated as patch versions, for example in the 6.15 multiple `export default` are throwing an error but before it was allowed.

Can you point me to which breaking change you are referring to?

**Comment by hillerstorm** [comment]
@danez https://github.com/babel/babel/commit/26e79c543374e99e6742eac69d8f1f9d4ecf5ed1#diff-3953cdd7a44c58e0431a2e202efb64d4L114
this commit changed the signature of the default export to require a `helpers` object, which is never supplied when coming from `babel-plugin-transform-async-to-generator` <6.16.0

In our case we had version 6.8.0 hardcoded and a coworker got this error on a fresh npm install while my code ran fine.
I realise the issue cannot be fixed other than upgrading to 6.16.0 since 6.8.0 will forever have ^6.8.0 specified as its dependency for the helper. Or we could add that exact version of the helper in our own package.json but it's still not a fun way to discover bugs :)
